### PR TITLE
Bump the apacheds docker version for ldap

### PR DESCRIPTION
**Pull request by pwalczysko**
bump the version of apacheds docker, as found during testing of https://github.com/openmicroscopy/prod-playbooks/pull/130 - the recent version is crucial for correct functionality

cc @joshmoore @manics

### CODE REVIEW

File: omero/training-server/playbook.yml [hunk 1]
```diff
@@ -319,7 +319,7 @@
     # You may wish to change this variable,
     # or override it by defining the private variable os_system_users_password_override.
     os_system_users_password: "{{ os_system_users_password_override | default('$6$leKi5B1PgSvdA/ec$xbU3CnoSFnYdeZjEjKK5TH8SGATsW746uopssff4edpgyu.cWXGo9A.oK6wH9kIkxLCCNcORGZnnroZPMqGzN/') }}"
-    apache_docker_release: "{{ apache_docker_release_override | default('0.3.0') }}"
+    apache_docker_release: "{{ apache_docker_release_override | default('0.6.0') }}"
```
Comment (manics) [inline omero/training-server/playbook.yml]: You could delete `apache_docker_release_override` too, so it's just `apache_docker_release: '0.6.0'`?

Comment (pwalczysko) [reply omero/training-server/playbook.yml]: But note that this is in line with the overall override system for all the variables ?
Is it advisable to have some of them with the override variable, some not ?
Definitely, some of the variables here must be overriden.

**Comment by pwalczysko** [comment]
>If you want to keep it as is this is good to merge.

I think I would like to keep it as it is.

**Comment by pwalczysko** [comment]
Regarding https://github.com/openmicroscopy/prod-playbooks/pull/132#discussion_r247466193 and my answer above, maybe 

Ready to merge ?

**Pull request merged**